common/util.go

```diff
@@ -86,3 +89,18 @@ func ReadSerializedJsonContent(jsonStr, filePath string) (serJSON string, md5Has
 	md5Hash = CalculateMd5Hash(content)
 	return string(content), md5Hash, nil
 }
+
+// Suppress the error if it is 404
+func IgnoreNotFoundError(err error) error {
+	var apiErr *apierr.APIError
+	if !errors.As(err, &apiErr) {
+		return err
+	}
+	if apiErr.StatusCode == 404 {
+		return nil
+	}
+	if strings.Contains(apiErr.Message, "does not exist") {
+		return nil
+	}
```

Comment:
Does this work? This will allow us to handle mapping of API responses to error codes consistently in the SDK, so we don't need to maintain custom logic in the TF provider.
```suggestion
	if errors.Is(err, databricks.ErrResourceDoesNotExist) {
		return nil
	}
```

Reply:
`strings.Contains(apiErr.Message, "does not exist")` is used for cases when it's not 404...